README.md

```diff
@@ -21,5 +21,13 @@ Check full features and entities involved on the [wiki home page](https://github
 8. Create database: `rake db:create`
 9. Run migrations: `rake db:migrate`
 10. Seed database with initial data: `rake db:seed`
+This creaete five users with 
```

Comment:
Users creation is not the only thing involved in seeding. Can you confirmed once again if we are only adding users or more data?